matroska_tags.xml

```diff
@@ -276,10 +276,10 @@
       <description lang="en">This is a binary dump of the TOC of the CDROM that this item was taken from. This holds the same information as the MCDI in ID3.</description>
     </tag>
     <tag name="ISBN" class="Identifiers" type="UTF-8">
-      <description lang="en">[International Standard Book Number](http://www.isbn-international.org/)</description>
+      <description lang="en">[International Standard Book Number](https://www.isbn-international.org/)</description>
     </tag>
     <tag name="BARCODE" class="Identifiers" type="UTF-8">
-      <description lang="en">[EAN-13](http://www.ean-int.org/) (European Article Numbering) or [UPC-A](http://www.uc-council.org/) (Universal Product Code) bar code identifier</description>
+      <description lang="en">[EAN-13](https://www.ean-int.org/) (European Article Numbering) or [UPC-A](http://www.uc-council.org/) (Universal Product Code) bar code identifier</description>
```

Comment:
Warning: Potential Security Risk Ahead
Firefox does not trust this site because it uses a certificate that is not valid for www.ean-int.org. The certificate is only valid for the following names: *.uniweb.be, uniweb.be

Reply:
Hmm… on my end, the original one now redirects to `https://www.gs1.org`.

Reply:
I tested on Firefox, Chrome, Edge on Windows.
Redirect to gs1 is only with the http version.
So here I suggest to change the URL completely to `https://www.gs1.org/`

Reply:
done